benchmarks/operator_benchmark/pt/qactivation_test.py

```diff
@@ -32,8 +32,13 @@
 )
 
 qactivation_short_configs = op_bench.cross_product_configs(
-    dims=((3, 4, 5),      # Rank=3
-          (2, 3, 4, 5)),  # Rank=4,
+    dims=(
+        (3, 4, 5),      # Rank=3
+        (2, 3, 4, 5),    # Rank=4,
+        # Dimensions from the floating point benchmarks
+        (512, 512),
+        (256, 1024),
+    ),
```

Comment:
Be careful with adding large benchmarks in the short_configs -- those are run every time, and if they are too slow, the test will take forever.

Reply:
makes sense.  The times to execute these are still reasonable so should be ok, compared to some more expensive short tags checked in (like fp batchnorm).